symbols/pdb/Test/Mono.Cecil.Pdb.Tests.csproj

```diff
@@ -1,30 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectGuid>{29300103-CB76-4A1D-B6FD-FFD91C1EC8AA}</ProjectGuid>
-    <RootNamespace>Mono.Cecil.Pdb.Tests</RootNamespace>
-    <AssemblyName>Mono.Cecil.Pdb.Tests</AssemblyName>
+    <TargetFrameworks>net40</TargetFrameworks>
+
+    <EnableDefaultItems>False</EnableDefaultItems>
```

Comment:
this is redundant?

Reply:
No, this property can be removed (defaulted) when the project is converted to blobs. It's a bit of a challenge with the current repository because some projects are located in parent folders of other projects, which forces the use of various exclusions.